### PR TITLE
Admin UI

### LINKED ISSUE #148: ObjectBox - Data Browser
Please add Data Browsing for testing and debugging.

### PR DESCRIPTION
**Pull request by vaind**
Closes #148 

* [x] wait for the next version of C-API to be released (all platforms) #372 
* [ ] update docs https://docs.objectbox.io/data-browser // prepared draft
* [x] Update change log.

### CODE REVIEW

File: objectbox/lib/src/native/browser.dart [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  @pragma('vm:prefer-inline')
+  Pointer<OBX_browser> get _ptr => (_cBrowser.address != 0)
+      ? _cBrowser
+      : throw StateError('Browser already closed');
+
+  /// Whether the loaded ObjectBox native library supports ObjectBrowser.
+  static bool isAvailable() => C.has_feature(OBXFeature.ObjectBrowser);
+
+  /// Creates a sync client associated with the given store and options.
+  /// This does not initiate any connection attempts yet: call start() to do so.
```
Comment (greenrobot-team) [inline objectbox/lib/src/native/browser.dart:26]: "sync client" and "call start()", forgot to update docs?

File: objectbox/pubspec.yaml [hunk 2]
```diff
@@ -53,7 +53,7 @@ ffigen:
         '[^_]+_(.*)': '$1'
   # add location to standard library imports, e.g. stddef.h
   llvm-lib: '/usr/lib/'
-  compiler-opts: '-I/usr/lib/clang/11.1.0/include'
+  compiler-opts: '-I/usr/lib/clang/12.0.1/include'
```
Comment (greenrobot-team) [inline objectbox/pubspec.yaml:56]: Changed on purpose? Edit: ah, this was for latest C-API, so probably yes.

File: objectbox/lib/src/native/browser.dart [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+/// The Browser runs directly on your device or on your development machine.
+/// Behind the scenes this works by bundling a simple HTTP browser into
+/// ObjectBox when building your app. If triggered, it will then provide a basic
+/// web interface to the data and schema.
+class Browser {
+  late Pointer<OBX_browser> _cBrowser;
+  late final Pointer<OBX_dart_finalizer> _cFinalizer;
+
+  @pragma('vm:prefer-inline')
+  Pointer<OBX_browser> get _ptr => (_cBrowser.address != 0)
```
Comment (greenrobot-team) [inline objectbox/lib/src/native/browser.dart:18]: Re-use `!isClosed()` here?

File: objectbox/lib/src/native/browser.dart [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        this, native_browser_close, _cBrowser.cast(), 1024 * 1024);
+    if (_cFinalizer == nullptr) {
+      close();
+      throwLatestNativeError();
+    }
+  }
+
+  /// Closes and cleans up all resources used by this ObjectBrowser.
+  void close() {
+    if (_cBrowser.address != 0) {
```
Comment (greenrobot-team) [inline objectbox/lib/src/native/browser.dart:59]: Re-use `!isClosed()` here?

**Review by greenrobot-team — COMMENTED** [review]
LGTM. Found a docs issue. Potential minor improvements.

File: objectbox/lib/src/native/browser.dart (hunk 1 above)
Comment (vaind) [reply objectbox/lib/src/native/browser.dart]: fixed

File: objectbox/lib/src/native/browser.dart (hunk 3 above)
Comment (vaind) [reply objectbox/lib/src/native/browser.dart]: ok

File: objectbox/lib/src/native/browser.dart (hunk 4 above)
Comment (vaind) [reply objectbox/lib/src/native/browser.dart]: ok

**Comment by greenrobot-team** [comment]
For docs, to override dependency for Android:
```
android/app/build.gradle

// Tell Gradle to exclude the objectbox-android dependency from the implementation configuration
// that was added by objectbox_flutter_libs.
configurations {
    implementation {
        exclude group: 'io.objectbox', module: 'objectbox-android'
    }
}

dependencies {
    // Manually add objectbox-android-objectbrowser only for debug builds,
    // keep using objectbox-android for release builds.
    def objectboxVersion = "3.1.1"
    debugImplementation("io.objectbox:objectbox-android-objectbrowser:$objectboxVersion")
    releaseImplementation("io.objectbox:objectbox-android:$objectboxVersion")
}
```

**Pull request merged**